x-pack/plugins/code/public/components/main/main.scss

```
@@ -279,3 +279,7 @@
   border-right: $euiBorderThick;
   width: 316px !important;
 }
+
+.code-history-pagination {
+  margin: 5px 0 5px 5px;
```

Can you use EUI variables here instead like:
```suggestion
  margin: $euiSizeXS 0 $euiSizeXS $euiSizeXS;
```

which would be `4px` instead of `5px` but the number itself seemed arbitrary?

---

Good catch! Thanks. Actually, I think we don't this css class anymore.